### ISSUE TITLE
[FEATURE] .d.ts

### ISSUE THREAD

**Issue by daniel007enirman** [uwla/vue-data-table#38]
```
Could not find a declaration file for module '@andresouzaabreu/vue-data-table'.

**Comment by AndreSouzaAbreu** [comment]
Hi. Can you give details of  the context of your problem?

**Comment by zwoefler** [comment]
I have a similar issue. 

**In my Editor VS Code** when I import your component as:
```javascript
import DataTable from "@andresouzaabreu/vue-data-table"
```
it prints the informational message next to the import statement.

```javascript
Could not find a declaration file for module '@andresouzaabreu/vue-data-table'. '/home/martin/Projects/praesens/praesens/node_modules/@andresouzaabreu/vue-data-table/dist/DataTable.umd.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/andresouzaabreu__vue-data-table` if it exists or add a new declaration (.d.ts) file containing `declare module '@andresouzaabreu/vue-data-table';`Vetur(7016)
```

**On my webpage** I receive the following error:
```javascript
Uncaught (in promise) TypeError: _vm._self is undefined
    render DataTable.umd.js:4744
```

I don't know where to go from here, would really love to use your component in Vue3

**Comment by AndreSouzaAbreu** [comment]
Are you guys both using Vue3?

This plugin currently doesn't work with Vue3. I haven't been able to figure out yet why is that.
It works perfectly fine with Vue2, though.

**Comment by zwoefler** [comment]
Thanks for taking the time to answer here. Yes, as stated I a using Vue3. 
Can't speak for @daniel007enirman though

**Comment by uwla** [comment]
This has been fixed in commitment d95393., available at `@uwlajs/vue-data-table@^2.0.1` or above.
